Dockerfile

```diff
@@ -41,11 +41,17 @@ RUN add-apt-repository -y ppa:ubuntugis/ubuntugis-experimental \
     libssl-dev \
     libudunits2-dev \
     libv8-3.14-dev \
+    libxtst6 \
```

Comment:
Wow! Is this the only extra-dependency required to get `decktape` working? I had a much [longer list](https://github.com/darribas/gds19/blob/master/content/infrastructure/Dockerfile#L4)! Can you confirm that with this extra one `decktape` compiles to reveal.js to pdf fine?